node/pkg/reporter/attestation_events_test.go

```diff
@@ -0,0 +1,34 @@
+package reporter
+
+import (
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+	"testing"
+)
+
+func TestEventListener(t *testing.T) {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+
+	subs_expect := map[int]*lifecycleEventChannels{}
+
+	attestationEventReporter := EventListener(logger)
+	assert.Equal(t, subs_expect, attestationEventReporter.subs)
+}
+
+func TestGetUniqueClientId(t *testing.T) {
+	logger, _ := zap.NewProduction()
+	defer logger.Sync()
+
+	attestationEventReporter := EventListener(logger)
+	assert.Equal(t, 498081, attestationEventReporter.getUniqueClientId())
```

Comment:
This seems like a solid test, but where does that number come from?

If you can swing it, [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) are better when they are set to more obvious variables describing what they do.